use std::io::{Error, Result, ErrorKind};

fn main() {
    action_one();
}

fn action_one() -> Result<()> {
    Ok(())
}
